src/zenml/metadata_stores/base_metadata_store.py

```diff
@@ -48,12 +49,10 @@
 logger = get_logger(__name__)
 
 
-class BaseMetadataStore(StackComponent, ABC):
+class BaseMetadataStore(BaseModel, ABC):
     """Base class for all ZenML metadata stores."""
 
-    # Class Configuration
-    TYPE: ClassVar[StackComponentType] = StackComponentType.METADATA_STORE
-
+    uuid: UUID = Field(default_factory=uuid4)
```

Comment:
Any reason for this UUID still existing? For some potential scoping inside the metadata store?

Reply:
I guess we can take it out. Initially I kept this since the post execution classes used it, but I refactored that out already. The only consideration, as you said, might be to keep it in case we need to have multiple metadata stores within the zenstore (which I think we want?). TL;DR not sure whether this can go or not :D

Reply:
Alright we can just keep it for now till we're sure how we handle scoped metadata stores I guess.